src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FruitDao;
+import core.basesyntax.db.impl.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CsvFileWriterService;
+import core.basesyntax.service.QuantityCalculatorService;
+import core.basesyntax.service.impl.CsvFileReaderServiceImpl;
+import core.basesyntax.service.impl.CsvFileWriterServiceImpl;
+import core.basesyntax.service.impl.QuantityCalculatorServiceImpl;
+import core.basesyntax.service.impl.TransactionParserServiceImpl;
+import core.basesyntax.strategy.impl.OperationHandlerStrategyImpl;
+import java.util.List;
+
+public class Application {
+    private static final String VALID_READ_FROM_FILE =
+            "src/main/resources/ValidReadFromFile.csv";
+    private static final String WRITE_TO_FILE =
+            "src/main/resources/WriteToFile.csv";
+
+    public static void main(String[] args) {
+        QuantityCalculatorService quantityCalculator = new QuantityCalculatorServiceImpl(

```

Comment:
let's try to introduce variable before the place where it used and try to write the code that can be readed easily line by line like a book. See the example:
```
List<String> linesFormFile = new CsvFileReaderServiceImpl()
                .readFormFile(VALID_READ_FROM_FILE);

List<FruitTransaction> fruitTransactions = new TransactionParserServiceImpl()
                .getTransactions(linesFormFile);

new QuantityCalculatorServiceImpl(new OperationHandlerStrategyImpl())
                .calculate(fruitTransactions);
        
CsvFileWriterService csvFileWriterService = new CsvFileWriterServiceImpl();
csvFileWriterService.writeToFile(WRITE_TO_FILE);
```